frontend/simple/js/state.js

```diff
@@ -135,6 +188,7 @@ const actions = {
       }
       // TODO: verify each entry is signed by a group member
       await db.addLogEntry(contractId, entry)
+      entry.data.hash = hash
```

Comment:
The only thing that gets to see that level is this function. So unless I pull that data into an entry object and rehash it I won't know the hash.